templates/Dockerfile.template

```diff
@@ -155,23 +149,14 @@ ENV PATH /usr/share/elasticsearch/bin:$PATH
 
 WORKDIR /usr/share/elasticsearch
 
-{{- if (eq .Repo "elasticsearch6") }}
 COPY config /etc/elasticsearch
-{{- else }}
-COPY config ./config
-{{- end }}
 
 RUN set -ex \
 	&& for path in \
 		./data \
 		./logs \
-{{- if (eq .Repo "elasticsearch6") }}
 		/etc/elasticsearch \
 		/etc/elasticsearch/scripts \
-{{- else }}
```

Comment:
Just confirming the {{else}} was for versions older than 6.x and 7.x has the same directory structure?

Reply:
Yes, else was for older versions only.